aggregators/will-get-add-to-homescreen-prompt/index.js

```diff
@@ -94,6 +98,11 @@ class AddToHomescreen extends Aggregate {
       weight: 0
     };
 
+    criteria[manifestShortNameLength] = {
+      value: true,
+      weight: 0
```

Comment:
yeah, I think that's great. I was actually all ready to say that I didn't think you should be docked in your score just because your name was a little long